pymc/data.py

```diff
@@ -482,6 +482,8 @@ class Data:
     export_index_as_coords: bool, optional, default=False
         If True, the `Data` container will try to infer what the coordinates should be
         if there is an index in `value`.
+    **kwargs: dict, optional
+        Extra arguments passed to Aesara ``SharedVariable``.
```

Comment:
`SharedVariable` should be a clickable link so users can easily find what values are accepted and what are not without needing to first get to aesara docs, then finding sharedvariable within it.

Taking a look at the code, it should be `` :func:`aesara.shared` ``

Reply:
Oh I totally forgot about [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html)! Good catch. I'll also apply this change to the top level `SharedVariable` reference. Apprecaited

Reply:
No problem, it is particularly tricky because you can't really trust the rest of the docstring to be following numpydoc. Hopefully we'll get there little by little

Reply:
- Rebased via 509a6e3 to address args reference
- Added 56c45fa for top level doc reference